agent.go

```diff
@@ -360,6 +360,14 @@ func (agent *agentS) setHost(host string) {
 	agent.host = host
 }
 
+func (agent *agentS) log() LeveledLogger {
```

Comment:
Minor: Inconsistency in the names of the getter functions

Reply:
I think this function was already removed in the following commits, wasn't it?